src/renderers/webgl/WebGLShadowMap.js

```diff
@@ -24,6 +24,11 @@ function WebGLShadowMap( _renderer, _objects, _capabilities ) {
 
 		_depthMaterial = new MeshDepthMaterial( { depthPacking: RGBADepthPacking } ),
 		_distanceMaterial = new MeshDistanceMaterial(),
+		_ditheredDepthMaterial = new MeshDepthMaterial( {
+			depthPacking: RGBADepthPacking,
+			ditherTransparency: true
+		} ),
+		_ditheredDistanceMaterial = new MeshDepthMaterial( { ditherTransparency: true } ),
```

Comment:
This should be `MeshDistanceMaterial`.

Reply:
I've review the PR today and apart from the above tiny issue, the code looks good 👍 !